packages/kit/src/core/adapt/prerender.js

```diff
@@ -191,6 +192,11 @@ export async function prerender({ cwd, out, log, config, build_data, fallback, a
 					log.warn(`${rendered.status} ${decoded_path} -> ${location}`);
 					writeFileSync(file, `<meta http-equiv="refresh" content="0;url=${encodeURI(location)}">`);
 					written_files.push(file);
+
+					const resolved = resolve(path, location);
+					if (resolved.startsWith('/')) {
```

Comment:
we've had bugs with this logic before and it's in a few different places like line 129 of `load_node`. I wonder if we should make a util function for it. I think this will need to be updated to match `load_node` (and I'm pretty sure it's somewhere else too  - maybe elsewhere in prerendering):

```
resolved.startsWith('/') && !resolved.startsWith('//')
```

Reply:
good catch, added a util. bonus — it's significantly faster than calling `startsWith`